### DISCUSSION TITLE
Save failed plays with a shortcut instead of only mouse

### DISCUSSION THREAD

**Discussion by JacksonChen666** [ppy/osu#23940] [Ideas] [answered]
(Search term used to search before creating new discussion: "save replay shortcut")

I play osu!lazer. Sometimes I fail, and usually I'd like to save the failed play.

The save replay button is a button that is only available with the mouse, not the keyboard. So when I play a keyboard only ruleset (osu!standard with autopilot and pretty much all other built-in rulesets), saving replays is like a chore.

My process for saving a failed play goes as follows:
1. Press escape (during the fail animation) to bring up the fail screen
2. Move my mouse to wake it up from sleep (it's a wireless mouse, didn't touch it while playing something like osu!catch)
3. Locate my mouse by moving it around
4. Get in control of my mouse again
5. Move my cursor to the save replay button
6. Try to click it, except I'll miss it
7. Keep trying to click it, but I keep missing it
8. Move my mouse again, this time properly moving it to the button
9. Click the save replay button

Here's a video of me playing without the mouse, then trying to save a replay:

https://github.com/ppy/osu/assets/50755746/c4a73b78-67be-46ae-a858-6dd09ca41b4a

As you can see, it's a lot of steps that also takes quite a bit of time.

**What I'd like** is a keyboard shortcut.

With a keyboard shortcut, it should be even faster to save a replay:
1. Press escape (during the fail animation) to bring up the fail screen
2. Locate the key
3. Press the key

Advantage is I can quickly save the replay without the fuss of a mouse.

osu!lazer 2023.614.1-lazer

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23967
